modules/javafx.web/src/main/native/Source/WTF/wtf/FileSystem.h

```diff
@@ -70,7 +70,7 @@ namespace FileSystemImpl {
 #if USE(GLIB) && !OS(WINDOWS)
 typedef GFileIOStream* PlatformFileHandle;
 const PlatformFileHandle invalidPlatformFileHandle = 0;
-#elif OS(WINDOWS)
+#elif OS(WINDOWS) && !PLATFORM(JAVA)
```

Comment:
This looks like a safe and non-intrusive fix. Longer term, it might be worth refactoring so we don't have to repeat the `PLATFORM(JAVA)` test, but that can be done later.

Reply:
Either changing OS(WINDOWS) to PLATFORM(WIN) or moving PLATFORM(JAVA) to the top would address @kevinrushforth's concerns :)

Reply:
The latter might be best, and can be done in a follow-up (the current fix is fine for now). Thanks.